src/test/java/com/cst438/EndToEndNewAssignment.java

```diff
@@ -0,0 +1,67 @@
+package com.cst438;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.junit.jupiter.api.Test;
+
+public class EndToEndNewAssignment {
+
+	@Test
+	public void testEndToEndAssignment() {
+		
+		// Set the path to the ChromeDriver executable
+		System.setProperty("webdriver.chrome.driver", "chromedriver.exe");
+
+		// Create a new instance of the ChromeDriver
+		WebDriver driver = new ChromeDriver();
+
+		// Navigate to the home page
+		driver.get("http://localhost:3000/");
+
+		// Click on the 'ADD NEW ASSIGNMENT' button
+		WebElement addAssignmentButton = driver.findElement(By.id("create-new-assignment"));
```

Comment:
When testing with the main branch on the gradebook frontend, I am getting this error:
`org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"css selector","selector":"#create\-new\-assignment"}
`
Did you update the frontend code for this assignment (specifically the id for the ADD NEW ASSIGNMENT button in Assignment.js)?

Reply:
Approved, works with the pushed frontend changes.